src/build/get_build_environment.mjs

```diff
@@ -40,9 +47,11 @@ export async function getBuildEnvironment(platform, buildMode, sentryDsn) {
     }
   }
 
+  const appVersion = candidateId.match(/\/?v?([[0-9]+\.[0-9]+\.[0-9]+)/)[MATCH_INDEX];
+
   return {
     SENTRY_DSN: sentryDsn,
-    APP_VERSION: `${await getVersion(platform)}${buildMode === 'debug' ? '-debug' : ''}`,
-    APP_BUILD_NUMBER: await getBuildNumber(platform),
+    APP_VERSION: buildMode === 'release' ? appVersion : `${appVersion}-${buildMode}`,
```

Comment:
No need to change in this PR, but it would be a lot clearer to me is we used `APP_VERSION_NAME` for the version string.